### PR TITLE
Refactor: Remove history v1

### PR DESCRIPTION
**Pull request by tmrdlt**
- Deleted `ImporterV1`, `ValidatorV1`, `HistoryV1Db`, `HistoryV1ErrorDb`
- Renamed `ImporterV2`, `ValidatorV2`, `HistoryV2Db`, `HistoryV2ErrorDb` to `Importer`, `Validator`, `HistoryDb`, `HistoryErrorDb`
- Kept history API but added TODOs to adjust later

### CODE REVIEW

File: src/api/pair-liquidity-info-history/pair-liquidity-info-history.controller.spec.ts [hunk 1]
```diff
@@ -34,93 +33,94 @@ describe('PairLiquidityInfoHistoryController', () => {
...
-            reserve0: historyEntry2.reserve0,
-            reserve1: historyEntry2.reserve1,
-          },
-          height: historyEntry2.height,
-          microBlockHash: historyEntry2.microBlockHash,
-          microBlockTime: historyEntry2.microBlockTime.toString(),
-        },
-      ]);
-    });
+    // TODO fix with updated return
```
Comment (kenodressel) [inline src/api/pair-liquidity-info-history/pair-liquidity-info-history.controller.spec.ts:43]: try `it.skip` :)

**Review by kenodressel — COMMENTED** [review]
Looks good in general. Can we take note of the things that we are skipping in this PR so we at least have some record of tech debt that we need to clean later?

**Comment by tmrdlt** [comment]
I created a task for the future tech dept which consists of the API migration:
https://yellowgreen-mindblowing-masters.awork.com/tasks/98f5ae54-2931-4bd8-831b-087a4ca1c30b/details

**Comment by tmrdlt** [comment]
I created a task for the tech debt which consists of the API migration.

**Review by kenodressel — APPROVED** [review]

**Pull request merged**